src/common/ceph_time.h

```diff
@@ -533,4 +533,9 @@ template<typename Rep, typename Period>
 ostream& operator<<(ostream& m, const chrono::duration<Rep, Period>& t);
 }
 
+#if FMT_VERSION >= 90000
```

Comment:
That's the only modified file I wasn't sure about:
I'd rather use the fmtlib in reverse: use fmtlib for operator<<

Reply:
i don't feel strong either way. but i'd like to focus on enabling ceph to build with fmt v9 first. and then try to reimplement the formatter.